docs/DafnyRef/Statements.md

```diff
@@ -322,20 +357,25 @@ These expressions are then evaluated, then they are
 assigned to the yield parameters, and then the iterator
 yields.
 
-## 20.6. Update and Call Statements {#sec-update-and-call-statement}
-````grammar
-UpdateStmt =
-    Lhs
-    ( {Attribute} ";"
-    |
-     { "," Lhs }
-     ( ":=" Rhs { "," Rhs }
-     | ":|" [ "assume" ]
-               Expression(allowLemma: false, allowLambda: true)
-     )
-     ";"
-    )
-````
+## 20.6. Update and Call Statements ([grammar](#g-update-and-call-statement)) {#sec-update-and-call-statement}
+
+Examples:
+<!-- %check-resolve -->
+```dafny
+class C { var f: int }
+method q(i: int, j: int) {}
+method r() returns (s: int, t: int) { return 2,3; }
+method m() {
+  var ss: int, tt: int, c: C?, a: array<int>;
```

Comment:
```suggestion
  var ss: int, tt: int, c: C?, a: array<int>, d: D?;
```

Reply:
What does this add, given there is `c: C?`

Reply:
OK